docs/demography.md

```diff
@@ -1514,16 +1514,109 @@ modelled in the oldest epochs.
 
 (sec_demography_importing)=
 
-## Importing model definitions
+## Importing/exporting model definitions
 
 (sec_demography_importing_demes)=
 
 ### Demes
 
-```{eval-rst}
-.. todo:: Details of how we import Demes models.
+The [Demes format](https://popsim-consortium.github.io/demes-spec-docs/)
+is intended to become a standard for writing demographic models,
+to facilitate interchange between simulators, as well as between
+demographic inference software and simulators.
+Demes models are typically written as a YAML file (see the
+[Demes tutorial](demes-spec:sec_tutorial) for details), and loaded into Python
+using the [demes](demes:sec_introduction) library.
+The {meth}`.Demography.from_demes` method converts a {class}`demes.Graph`
+object into a {class}`.Demography` object, and the {meth}`.Demography.to_demes`
+method does the opposite conversion.
+
+```{warning}
+Demes uses similar terminology to msprime, but uses different conventions.
+In particular, Demes uses forwards-time conventions.
```

Comment:
"as opposed to the {ref}`backwards-time <sec_demography_direction_of_time>` conventions used in msprime" or such. Good to link back to the clarifying section.